Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -41,6 +41,8 @@ int main()
     cout << "Actual value of y: " << y << endl;
     
     MySwap(p, &y);
```

Comment:
I know this will work but the instruction specified to create an `integer pointer you make that points to the variable y`. Then you can pass that pointer to `MySwap()`.